core/dbt/parser/schema_yaml_readers.py

```diff
@@ -669,16 +669,25 @@ def _generate_saved_query_config(
 
         return config
 
-    def _get_export_config(self, unparsed: UnparsedExportConfig) -> ExportConfig:
-        return ExportConfig(
-            export_as=unparsed.export_as,
-            schema_name=unparsed.schema,
-            alias=unparsed.alias,
+    def _get_export_config(
+        self, unparsed: Dict[str, Any], saved_query_config: SavedQueryConfig
```

Comment:
super-nit: perhaps `unparsed` could be renamed to `unparsed_export_config`? might make more clear what we're combining to get an export config.

Reply:
Absolutely 🙂